completion/bash/task.bash

```diff
@@ -1,26 +1,48 @@
-#!/bin/bash
+# vim: set tabstop=4 shiftwidth=4 expandtab:
```

Comment:
Isn't `#!/bin/bash` really needed here?

What does this vim syntax actually does here?

Reply:
The shebang line `#!/bin/bash` informs the (UNIX-like) system with which interpreter to run a script provided it is made executable. However, those bash completion script fragments are designed to be included (sourced) into an already running interpreter. The bash completion framework does so automatically on demand. There's no point in running them on their own. Hence, the shebang is useless.

The other line configures editors like vim (and compatible) how to format (indent) this source code. Other bash completions contain that as well.

Reply:
@MarioSchwalbe OK, last question: can we keep 2 space indentation as before? That's our convention for non-Go code as you can see on the `.editorconfig` file.

Reply:
The bash completion standard is actually 4 ;-), but I don't really care. I'm going to push a new commit.